yb-voyager/cmd/exportData.go

```diff
@@ -1600,9 +1600,9 @@ func finalizeTableAndColumnList(finalTableList []sqlname.NameTuple) ([]sqlname.N
 		}
 	}
 
```

Comment:
I was thinking - We are querying the source for same information (table columns and udts etc.) again in these two functions FilterUnsupportedTables and GetColumnsWithUnsupportedTypes
so maybe something along the lines
1. Fetch all the table column information like table->columns ( datatype, arrayTypeOrNot )
2. Filter unsupported tables using the above information
3. Filter unsupported columns
what do you think?

Reply:
Yeah we could do this as a next step(refactoring) and yes it will be a bit more optimized.

But for now i would like to stick with current implementation, it already tested and an incremental optimization over the older implementation.